shard.yml

```diff
@@ -12,7 +12,6 @@ crystal: ">= 1.0"
 dependencies:
   markd:
     github: icyleaf/markd
-    commit: 5e5a75d13bfdc615f04cc7ab166ee279b3b996d3
```

Comment:
I'd suggest using commit pinning for security, like for the other dependencies.

Reply:
New releases can include breaking changes, so keeping it unrestricted might break the builds after running `shards update`. Since the code is checked into the repo, that's not a big issue, just noting.

Reply:
Yes, but if you don't try, you could never know. If we realize a compatibility issue, we can add an appropriate restriction.